src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -115,7 +115,22 @@ <h4 class="panel-title  border-bottom-small  visible" translate>{{'YOUR_INFORMAT
 
       </div>
 
-      <div class="row" ng-if="$ctrl.donorDetails['donor-type'] === 'Household'">
+      <h4 class="panel-title  border-bottom-small  visible" ng-if="$ctrl.donorDetails['donor-type'] === 'Household'" translate>{{'SPOUSE_NAME'}}</h4>
+
+      <div class="row">
+        <div class="col-xs-12">
+          <button
+            type="button"
+            class="btn pull-left add-spouse-btn"
+            ng-class="{ 'btn-default': $ctrl.showSpouseFields, 'btn-primary': !$ctrl.showSpouseFields }"
+            ng-click="$ctrl.toggleSpouseFields()"
+            ng-if="$ctrl.donorDetails['donor-type'] === 'Household'"
+            translate
+          >{{$ctrl.showSpouseFields ? "Remove spouse" : "Add spouse"}}</button>
+        </div>
+      </div>
```

Comment:
We should probably check if it is ok to modify branded checkout like this. I'm guessing they will want to only do this for the sign up process, not during checkout and branded checkout which is what is being done here.